examples/context-propagation/manual/build.gradle

```diff
@@ -4,6 +4,7 @@ plugins {
 
 dependencies {
     implementation project(':core')
+    implementation 'com.spotify:completable-futures'
```

Comment:
Is it fine to pull this in? Should we remove this dependency as well?

Reply:
I tried to make the `context-propagation/dagger` example without it, but it doesn't look good as it is.
https://github.com/line/armeria/blob/master/examples/context-propagation/manual/src/main/java/example/armeria/contextpropagation/manual/MainService.java#L93

So I can use the dependency in this case, but if it's a problem, let me know. I will remove it. 😄

Reply:
No worries, then. 😄